overlord/snapstate/handlers.go

```diff
@@ -166,8 +169,23 @@ func (m *SnapManager) doPrerequisites(t *state.Task, _ *tomb.Tomb) error {
 		prereqName = snapsup.Base
 	}
 
+	if err := m.installPrereq(t, prereqName, snapsup.UserID); err != nil {
```

Comment:
A comment here would be nice. Is this trying to get prerequisites and prerequisites of prerequisites?